torch/nn/modules/batchnorm.py

```diff
@@ -25,15 +25,21 @@ def __init__(self, num_features, eps=1e-5, momentum=0.1, affine=True,
         if self.track_running_stats:
             self.register_buffer('running_mean', torch.zeros(num_features))
             self.register_buffer('running_var', torch.ones(num_features))
+            self.register_buffer('num_batches_tracked', torch.LongTensor([0]))
```

Comment:
Will this break backwards compatibility? I think you may not be able to load old state_dicts because they will be missing `num_batches_tracked`?

Reply:
yes this will. Which is why the PR was merged and reverted.
I think we should have a mechanism to plug-in backwards-compatibility for load_state_dict (we dont at the moment, unlike for standard pickling via `__setstate__`

Reply:
Please use `torch.tensor(0, dtype=torch.long)`